api.md

```diff
@@ -170,6 +171,118 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## add_data_node() [](add_data_node)
+
+Add a new data node to the current database for use with distributed
+hypertables.
+
+This includes bootstrapping the data node by creating and configuring
+a new database on it. Newly created distributed hypertables will be
+able to make use of this data node for distributing data across the
+cluster.
+
+Note, however, that already existing distributed hypertables may
+require a separate `attach_data_node()` call to schedule the data node
+for use with those tables as they do not automatically expand to
+include the new data node.
+
+The function will attempt to bootstrap the data node by creating the
+database where the remote hypertables will later be created and also
+load the TimescaleDB extension on the data node.
```

Comment:
Ah, I like this explanation of bootstrapping here, so definitely think we can remove the bootstrapping reference above

Reply:
Done.